code/modules/recycling/disposal-construction.dm

```diff
@@ -170,10 +170,13 @@
 			nicetype = "pipe"
 
 	var/turf/T = loc
-	if(T.intact)
+	if(T.intact && istype(T, /turf/simulated/floor))

```

Comment:
this probably lets you build disposal pipes on space and silly stuff like that. may want to make sure T is a simulated turf.